examples/iam-assumable-role/main.tf

```diff
@@ -127,16 +130,6 @@ data "aws_iam_policy_document" "custom_trust_policy" {
       identifiers = ["*"]
     }
   }
-
-  statement {
-    effect  = "Deny"
-    actions = ["sts:AssumeRole"]
-
-    principals {
-      type        = "AWS"
-      identifiers = ["arn:aws:iam::111111111111:root"]
```

Comment:
when testing the example, this failed due to the account ID (I assume) so I removed